frontend/src/metabase/entities/users.js

```diff
@@ -80,50 +85,74 @@ const Users = createEntity({
   },
 
   objectActions: {
-    resentInvite: async ({ id }) => {
-      MetabaseAnalytics.trackStructEvent("People Admin", "Resent Invite");
-      await UserApi.send_invite({ id });
-      return { type: RESEND_INVITE };
-    },
-    resetPasswordEmail: async ({ email }) => {
-      MetabaseAnalytics.trackStructEvent(
-        "People Admin",
-        "Trigger User Password Reset",
-      );
-      await SessionApi.forgot_password({ email });
-      return { type: PASSWORD_RESET_EMAIL };
-    },
-    resetPasswordManual: async ({ id }, password = generatePassword()) => {
-      MetabaseAnalytics.trackStructEvent(
-        "People Admin",
-        "Manual Password Reset",
-      );
-      await UserApi.update_password({ id, password });
-      return { type: PASSWORD_RESET_MANUAL, payload: { id, password } };
-    },
-    deactivate: async ({ id }) => {
-      MetabaseAnalytics.trackStructEvent("People Admin", "User Removed");
-      // TODO: move these APIs from services to this file
-      await UserApi.delete({ userId: id });
-      return { type: DEACTIVATE, payload: { id } };
-    },
-    reactivate: async ({ id }) => {
-      MetabaseAnalytics.trackStructEvent("People Admin", "User Reactivated");
-      // TODO: move these APIs from services to this file
-      const user = await UserApi.reactivate({ userId: id });
-      return { type: REACTIVATE, payload: user };
-    },
+    resetPasswordEmail:
+      ({ email }) =>
+      async dispatch => {
+        MetabaseAnalytics.trackStructEvent(
+          "People Admin",
+          "Trigger User Password Reset",
+        );
+        await await entityCompatibleQuery(
+          { email },
+          dispatch,
+          sessionApi.endpoints.forgotPassword,
+        );
+        dispatch({ type: PASSWORD_RESET_EMAIL });
+      },
+    resetPasswordManual:
+      async ({ id }, password = generatePassword()) =>
```

Comment:
Is this function `async`?

Reply:
It was/is
https://github.com/metabase/metabase/pull/41312/commits/4bf0e0e504881a2b9fc29eda6761359bb138cf99#diff-4f46204c4fd684bc8b51d8c6f9a2d791ce065787b6de8dce6808772d1d02f5c8L103